### DISCUSSION TITLE
Question about Navigation

### DISCUSSION THREAD

**Discussion by devanlooches** [pimalaya/himalaya-vim#5] [Q&A]
Hello,
I wanted to say that I just started looking at your plugin and I already replaced neomutt with it. :)
One quick question about navigation within neovim though.

When you switch to a Himilaya page (i.e. the Himalaya command, or previewing a message), How do I go back to the previous page?

Thank you for this wonderful plugin and the help!

**Comment by soywod** [comment]
> I wanted to say that I just started looking at your plugin and I already replaced neomutt with it. :)

Wow, I feel honoured and scared at the same time :rofl: keep in mind that the project is still very young, I would not advice to use in production before the `v1.0.0`.

> How do I go back to the previous page?

See Himalaya's views/pages as Vim buffers. So there is not really a "navigation". For example, when you list your mails with the `:Himalaya` command and then you preview a mail, the best way to come back to this list is either to kill your preview buffer (`:bd`) or to start again the `:Himalaya` command.
